Source/Core/Core/AchievementManager.h

```diff
@@ -187,6 +201,7 @@ class AchievementManager
 
   std::unordered_map<AchievementId, UnlockStatus> m_unlock_map;
   std::unordered_map<AchievementId, LeaderboardStatus> m_leaderboard_map;
+  std::map<std::string, std::unique_ptr<OSD::Icon>> m_active_challenges;
```

Comment:
This should likely have an alias to make usages a little less noisy (e.g. in functions above).

Also consider adding `std::less<>` to it, like:

```cpp
using CoolType = std::map<std::string, std::unique_ptr<OSD::Icon>, std::less<>>;
```

Which allows certain operations not require allocating a std::string (e.g. a string_view can be passed into find, etc).